controllers/job_controller.js

```diff
@@ -5,78 +5,75 @@ const controller = {
   async showJobPage(req, res) {
     try {
       const jobId = req.params.job_id;
-      const job = await Jobs.findById(jobId).populate("author_id");
+      const job = await Jobs.findByIdAndUpdate(jobId, { $pull: { photos: null, skills: null } }).populate("author_id");
       res.render("./jobs/index.ejs", { job });
     } catch (error) {
       console.log(error.message);
     }
   },
-  // //Method GET
-  // newProjectForm(req, res) {
-  //   res.render("./projects/project_new.ejs");
-  // },
+  //Method GET
+  newJobForm(req, res) {
+    res.render("./jobs/job_new.ejs");
+  },
 
-  // //Method POST: to create new project
-  // async create(req, res) {
-  //   const projectData = req.body;
-  //   const profileOwner = await Users.findById(`${req.session?.currentUser?._id}`);
-  //   projectData.author_id = profileOwner._id;
-  //   const newProject = await Projects.create(projectData);
-  //   res.redirect(`/projects/${newProject._id}`);
-  // },
+  //Method POST: to create new job
+  async create(req, res) {
```

Comment:
please check everywhere. don't just check at one point. this method and others too.

remember rule #1: consistency